### PR TITLE
added @ghuntley

### PR DESCRIPTION
**Pull request by ghuntley**
👋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -459,6 +460,18 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
...
+#### What Geoff Streams
+- Haskell, Reactive Extensions, NixOS, OSS Development, C#, .NET, Xamarin, DevOps
+#### Streaming on:
+- [Twitch](https://www.twitch.tv/geoffreyhuntley)
+- [Twitter](https://twitter.com/geoffreyhuntley)
+- [YouTube](https://www.youtube.com/c/geoffreyhuntley)
+#### Links:
+- [Website](https://www.ghuntley.com)
+- [GitHub](https://github.com/ghuntley)
+- [Patreon](https://www.patreon.com/geoffreyhuntley)
```
Comment (mblayman) [inline README.md]: This link took me to a 404 page.

File: README.md [hunk 2]
```diff
@@ -459,6 +460,18 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Pidgin](https://www.pidgin.im)
 - [Convey](https://bitbucket.org/rw_grim/convey)
 
+### Geoffrey Huntley
+#### What Geoff Streams
+- Haskell, Reactive Extensions, NixOS, OSS Development, C#, .NET, Xamarin, DevOps
+#### Streaming on:
+- [Twitch](https://www.twitch.tv/geoffreyhuntley)
+- [Twitter](https://twitter.com/geoffreyhuntley)
```
Comment (mblayman) [inline README.md]: Could you move Twitter down to the links section? Twitter doesn't do any kind of streaming, right?

File: README.md [hunk 3]
```diff
@@ -495,7 +508,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 
 ### Holden Karau
 #### What Holden Karau streams
-Scala, Python, Spark, PySpark, Open Source#### Streaming on:
```
Comment (mblayman) [inline README.md]: Thanks for cleaning that up!

File: README.md (hunk 2 above)
Comment (ghuntley) [reply README.md]: image

Twitter supports streams. See above. I simulcast on Twitch, Twitter and YouTube.

File: README.md (hunk 1 above)
Comment (ghuntley) [reply README.md]: Resolved. Thank-you!

**Comment by ghuntley** [comment]
Ready to go!

File: README.md (hunk 2 above)
Comment (mblayman) [reply README.md]: Wow, I had no idea they did that. Thanks for educating me!

**Pull request merged**